src/System.Management.Automation/resources/InternalCommandStrings.resx

```diff
@@ -165,4 +165,16 @@
   <data name="ValueNotSpecifiedForWhereObject" xml:space="preserve">
     <value>The specified operator requires both the -Property and -Value parameters. Provide values for both parameters, and then try the command again.</value>
   </data>
+  <data name="PSTaskStreamWriterWrongThread" xml:space="preserve">
+    <value>This method cannot be run on the current thread. It can only be called on the cmdlet thread.</value>
+  </data>
+  <data name="ParallelUsingVariableCannotBeScriptBlock" xml:space="preserve">
+    <value>A ForEach-Object -Parallel using variable cannot be a script block. Script blocks have undefined behavior when used in parallel.</value>
```

Comment:
I think it is important to provide an explanation as to why script blocks are not allowed to be passed in, especially since it is easy to get around the check and pass in script blocks objects indirectly.  I feel users should know that this scenario is not supported.  But I agree that the second sentence is not very clear ... I'll try to make it better.